src/riot-rs-runqueue/src/runqueue.rs

```diff
@@ -8,8 +8,23 @@ use self::clist::CList;
 const USIZE_BITS: usize = mem::size_of::<usize>() * 8;
 
 /// Runqueue number.
-pub type RunqueueId = u8;
-pub type ThreadId = u8;
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct RunqueueId(pub u8);
```

Comment:
Yes, I thought about that as well. I was leaning towards leaving it as it is, because `riot-rs-runqueue` doesn't really know anything about thread priorities and only cares about runqueues.
But then again, if we'd want to be consistent and have the naming in `riot-rs-runqueue` completely abstract, it should also be called `ItemId` instead of `ThreadId`.

So yes, I am in favor of renaming it to `ThreadPriority`. cc @kaspar030  wdyt?

Reply:
fine with me.
maybe split out into a different PR. your call.

Reply:
:+1:. Will do a different PR for it after this is merged.